### PR TITLE
104 babel runtime

### PR DESCRIPTION
**Pull request by menor**

### CODE REVIEW

File: scripts/copy-files.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  return new Promise((resolve) => {
+    fse.readFile(path.resolve(__dirname, '../package.json'), 'utf8', (err, data) => {
+      if (err) {
+        throw err;
+      }
+
+      resolve(data);
+    });
+  })
+  .then((data) => JSON.parse(data))
```
Comment (heinburger) [inline scripts/copy-files.js]: does JSON.parse return a promise?

File: scripts/icon-index-generator.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ rrs(svgIconPath).forEach((file) => {
         const moduleName = fileLines[index].split(' ')[2].replace(';', '').trim();
         const modulePath = file.substring(0, file.length - 3).replace(/\\/g, '/').replace(svgIconPath, '');
 
-        outArray.push(`export ${moduleName} from './${modulePath}';\n`);
+        outArray.push(`export ${moduleName} from './${modulePath}'\n`);
```
Comment (heinburger) [inline scripts/icon-index-generator.js]: this is pretty cool!

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,22 @@
-export * from './components'
-export * from './icons'
-export * from './elements'
-
-// variables
-export { colors, breakpoints } from './variables'
+// Components
```
Comment (heinburger) [inline src/index.js]: was this manual? Can we add a script similar to the image one for this? That would be awesome

Comment (menor) [inline src/index.js]: yes, can we leave it for 2.1 ? 🙏😅

**Pull request merged**